libs/remix-tests/src/runTestSources.ts

```diff
@@ -5,7 +5,7 @@ import { deployAll } from './deployer'
 import { runTest } from './testRunner'
 
 import Web3 from 'web3'
-import { Provider } from '@remix-project/remix-simulator'
```

Comment:
these changes concern `runTestSources`, should we do the same for `runTestFiles`?

Reply:
I am not sure if external import works fine with the `runTestFiles`

Reply:
which resolver is it used in that case?

Reply:
Probably none